lib/display.go

```diff
@@ -246,3 +221,26 @@ func certEnd(end time.Time) string {
 func redify(text string) string {
 	return red.SprintfFunc()("%s", text)
 }
+
+func oidify(oid asn1.ObjectIdentifier) string {
+	raw := oid.String()
+	names := map[string]string{
+		"2.5.4.3":                  "CommonName",
+		"2.5.4.5":                  "Serial Number",
```

Comment:
I think that seems fine - it doesn't really make sense for non-EV certs, but I don't think I've ever seen 2.5.4.5 used otherwise.